src/Microsoft.Health.Fhir.Core/Features/SupportedSortParameterNames.cs

```diff
@@ -0,0 +1,15 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+namespace Microsoft.Health.Fhir.Core.Features
+{
+    /// <summary>
+    /// Provides list of supported sort parameter names.
+    /// </summary>
+    public static class SupportedSortParameterNames
```

Comment:
This is sql-specific, so please put this in the SQL project

Reply:
Moved. The consequence of moving this requires to initialise the supported list when the SQL service starts.